File: server/src/main/java/domain/WorkExperience.java
package domain;

import orm.workExperience.WorkExperienceDO;

public class WorkExperience {
    private String businessName;
    private String startYear;
    private String endYear;

    public WorkExperience(String businessName, String startYear, String endYear) {
        this.businessName = businessName;
        this.startYear = startYear;
        this.endYear = endYear;
    }

    public WorkExperienceDO workExperienceToDO(String userEmail) {
        return new WorkExperienceDO(
                businessName,
                startYear,
                endYear,
                userEmail
        );
    }

    public String getBusinessName() {
        return businessName;
    }

    public String getStartYear() {
        return startYear;
    }

    public String getEndYear() {
        return endYear;
    }
}

File: client-app/lib/models/work_experience.dart
class WorkExperience {
  String startYear;
  String endYear;
  String businessName;
  
  WorkExperience({required this.startYear, required this.endYear, required this.businessName});

  WorkExperience.fromJson(Map<String, dynamic> json)
      : startYear = json['startYear'],
        endYear = json['endYear'],
        businessName = json['businessName'];

  Map<String, dynamic> toJson() {
    return {
      'startYear': startYear,
      'endYear': endYear,
      'businessName': businessName,
    };
  }
}